__tests__/app.test.js

```diff
@@ -0,0 +1,67 @@
+const request = require(`supertest`);
+const app = require(`../app.js`);
+const seed = require(`../db/seeds/seed.js`);
+const db = require(`../db/connection.js`);
+const testData = require(`../db/data/test-data/index.js`)
+const { TestWatcher } = require("jest");
+
+beforeEach(() => {
+    return seed(testData)
+})
+
+afterAll(() => {
+    return db.end()
+})
+
+describe (`/api/topics`, () => {
+    test(`GET - 200: returns an array of objects`, () => {
+        return request(app)
+            .get(`/api/topics`)
+            .expect(200)
+            .then((res) => {
+                expect(Array.isArray(res.body)).toEqual(true);
+            })
+    })
```

Comment:
This is essentially covered by your next test as you run a forEach on the body

Reply:
Removed as requested!